test/search/test_organized_index.cpp

```diff
@@ -49,6 +48,14 @@ using namespace pcl;
 
 std::string pcd_filename;
 
+// Here we want a very precise distance function. Speed is less important.
```

Comment:
geometry.hpp has this exact definition (but in float). If the precision is the issue, let's re-write the comment to specify that

Reply:
Do you mean [these](https://github.com/PointCloudLibrary/pcl/blob/master/common/include/pcl/common/distances.h#L181-L207)?
What do you think about `Here we want a very exact distance function with double precision. Speed is less important.`

Reply:
I meant [these](https://github.com/PointCloudLibrary/pcl/blob/master/common/include/pcl/common/geometry.h#L58-L72). Sadly, all of these are single precision.

Regarding the wording, some contrast wrt existing library code would be nice like `using double precision unlike pcl/common`

Reply:
I changed the comment, let me know what you think